packages/paste-website/src/components/word-list/WordFilter.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Label} from '@twilio-paste/label';
+import {Input} from '@twilio-paste/input';
+import {useUID} from '@twilio-paste/uid-library';
+import {Button} from '@twilio-paste/button';
+import {FilterIcon} from '@twilio-paste/icons/esm/FilterIcon';
+import {ClearIcon} from '@twilio-paste/icons/esm/ClearIcon';
+
+type WordFilterProps = {
+  value: string;
+  onInput: (e: React.FormEvent<HTMLInputElement>) => void;
+  onClear: () => void;
+};
+export const WordFilter: React.FC<WordFilterProps> = ({value, onInput, onClear}) => {
+  const filterID = useUID();
+  return (
+    <Box marginBottom="space70" maxWidth={['100%', '100%', 'size30']}>
+      <Label htmlFor={filterID}>Filter words:</Label>
```

Comment:
nitpick: I think our guidance says to not put colons at the end of labels (though I can't find anything saying that now).

Reply:
done